src/System.Buffers.Experimental/System/Buffers/BufferExtensions.cs

```diff
@@ -49,13 +49,13 @@ public static int CopyTo(this IMemoryList<byte> list, Span<byte> destination)
             return copied;
         }
 
-        public static Position? PositionOf(this IMemoryList<byte> list, byte value)
+        public static Collections.Sequences.Position? PositionOf(this IMemoryList<byte> list, byte value)
```

Comment:
Why is this namespace qualified?

Reply:
Because `Position` in buffer primitives it renamed `ReadCursor`, and this method returns the one from `Sequences`

Reply:
yeah, I noticed.